packages/@okta/vuepress-site/docs/reference/api/hook-keys/index.md

```diff
@@ -0,0 +1,360 @@
+```
+
+##### Response example
+
+```json
+{
+  "id": "HKY1p7jWLndGQV9M60g4",
+  "keyId": "7fbc27fd-e3df-4522-86bf-1930110256ad",
+  "name": "My new key",
+  "created": "2022-08-31T18:09:58.000Z",
+  "lastUpdated": "2022-08-31T18:09:58.000Z",
+  "isUsed": false,
+  "_embedded": {
+    "kty": "RSA",
+    "alg": "RSA",
+    "kid": "7fbc27fd-e3df-4522-86bf-1930110256ad",
+    "use": null,
+    "e": "AQAB",
+    "n": "2naqCnv6r4xNQs7207lRtKQvdtnlVND-8k5iYBIiqoKGY3CqUmRm1jleoOniiQoMkFX8Wj2DmVqr002efF3vOQ7_gjtTatBTVUNbNIQLybun4dkVoUtfP7pRc5SLpcP3eGPRVar734ZrpQXzmCEdpqBt3jrVjwYjNE5DqOjbYXFJtMsy8CWE9LRJ3kyHEoHPzo22dG_vMrXH0_sAQoCk_4TgNCbvyzVmGVYXI_BkUnp0hv2pR4bQVRYzGB9dKJdctOh8zULqc_EJ8tiYsS05YnF7whrWEyARK0rH-e4d4W-OmBTga_zhY4kJ4NsoQ4PyvcatZkxjPO92QHQOFDnf3w"
+  }
+}
+```
+
+### List keys
+
+<ApiOperation method="get" url="/api/v1/hook-keys" />
+
+Returns a list of keys
+
+##### Request examples
+
+```bash
+curl -v -X GET \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/hook-keys"
+```
+
+##### Response example
+
+```json
+[{
+  "id": "HKY1i2htmXF5UNQhL0g4",
+  "keyId": "bb5bed7d-6e4d-488f-9c86-59b93a2bb3fb",
+  "name": "My new key",
+  "created": "2022-08-22T16:34:33.000Z",
+  "lastUpdated": "2022-08-22T16:34:33.000Z",
+  "isUsed": true
+}, {
+  "id": "HKY1p7jWLndGQV9M60g4",
+  "keyId": "7fbc27fd-e3df-4522-86bf-1930110256ad",
+  "name": "Test Key",
+  "created": "2022-08-31T18:09:58.000Z",
+  "lastUpdated": "2022-08-31T18:09:58.000Z",
+  "isUsed": false
+}]
+```
+
+### Get a public key
+
+<ApiOperation method="get" url="/api/v1/hook-keys/${keyId}" />
+
+Returns the public portion of the key object using the `keyId` parameter.
+
+##### Request parameters
+
+| Parameter | Description               | Param Type   | DataType   | Required |
+| --------- | ------------------------- | ------------ | ---------- | -------- |
+| `keyId`      | A valid key ID   | Path         | String     | TRUE     |
+
+> **Note:** `keyId` is the alias of the public key.
+
+##### Response parameters
+
+The response represents the key that matches the `keyId` you specify.
+
+##### Request example
+
+```bash
+curl -v -X GET \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/hook-keys/${KeyId}"
+```
+
+##### Response example
+
+The details of the response can be found in [public key details](#public-key-details).
+
+```json
+{
+  "kty": "RSA",
+  "alg": "RSA",
+  "kid": "bb5bed7d-6e4d-488f-9c86-59b93a2bb3fb",
+  "use": null,
+  "e": "AQAB",
+  "n": "togCavBd7yQ7fuTEdvRURfK04LSgxqbKlptc7Gy9l6Swk2nhJbfIuB2Eezrn_JX-Azh-8usMMKau8vkq6AR2pW5mgDNIJG0OnCdwK24BdohxWlVQRZ5fD5DYg-noWwIbDjCGxMq3BKhOJP8cGAXx-olfrkC2NRy-HiJZOPiONII_GnlBiVS29BmULbbhKNeSGtoe90tR_OsXozfBsRRKFnF4ewQG3QwE23q6W3uNsUTr87fD9OO0FxNXyjnR3UrMTq36W6QLhqgLxlSiqxXGALnhK5z7WfXxyGfYrytrg2tFIhiHi0hT300xRI4A6SqhPFSCegx_O8iZQlTeaiHgjQ"
+}
+```
+
+### Update a key
+
+<ApiOperation method="put" url="/api/v1/hook-keys/${id}" />
+
```

Comment:
Replaces a key

I'm following Royal's guidelines for Redocly wording going forward